java/hive/src/main/resources/META-INF/services/org.apache.hadoop.hive.ql.io.StorageFormatDescriptor

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020-2023 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Register vineyard filesystem to hadoop-fs, see also:
```

Comment:
I think you need to revise the comment here, as you are not registering file system.

Be careful when you do copy-paste.

Reply:
okk.I'll be more careful future.